src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -6,26 +6,24 @@
 import os
 import json
 import time
-import subprocess
 from subprocess import Popen, PIPE
 from base64 import b64encode
 
 from knack.util import CLIError
 from knack.log import get_logger
 from azure.cli.core.commands.client_factory import get_subscription_id
 from azure.cli.core.util import sdk_no_wait
-from azure.cli.core._profile import Profile
 from azure.cli.core import telemetry
+from msrestazure.azure_exceptions import CloudError
+from kubernetes import client as kube_client, config
+from Crypto.IO import PEM
```

Comment:
this is a 3rd party library? how does it packaged into your extension?

Reply:
It is being imported from the dependency 'pycryptodome' present in setup.py